src/atoms/Button/__tests__/button.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Button from '../index';
+import sinon from 'sinon';
+import assign from 'lodash/assign';
+import { expect } from 'chai';
+
+describe('Button', () => {
+  let defaultProps;
+
+  beforeEach(() => {
+    defaultProps = {
+      style: 'button',
+      onClick: sinon.spy(),
+      className: 'button',
+      shake: false,
+      type: 'button',
+      disabled: false
+    };
+  });
+
+  it('is a <button> tag', () => {
+    const wrapper = shallow(<Button />);
+    expect(wrapper.type()).to.equal('button');
+  });
+
+  it('adds the style prop as a className', () => {
+    const props = assign({}, defaultProps, {style: 'info'});
```

Comment:
@jmcolella ahh ok - I'll hold off merging this until your PR is ready, and then I'll switch over to using `kind` or whatever we decide to use